css/product-payment.css

```diff
@@ -572,4 +572,302 @@
   .product-page__add-target{
     margin-left: 180px;
   }
-}
\ No newline at end of file
+}
+
+/* Movil style */
+.improve-purchase{
+  max-width: var(--max-widthM);
+}
+
+.improve-purchase__top{
+  display: flex;
+  gap: var(--font-size-small);
+  margin-left: 20px;
+}
+
+.improve-purchase__buttom {
+  border: none;
+  color: var(--color-gray);
+  text-align: center;
+  border-radius: var(--font-size-smalls);
```

Comment:
El diseño dice que el valor es 8px, no 14px

<img width="430" alt="image" src="https://github.com/user-attachments/assets/da136856-9749-487c-930f-94937b142a27">

```suggestion
  border-radius: var(--eight-pixel);
```